{"name": "Try to update pet with invalid status", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 405. Actual: 200", "trace": "self = <tests.test_update_pet.TestUpdatePet object at 0x106306ad0>\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.title(\"Try to update pet with invalid status\")\n    def test_update_pet_with_invalid_status(self):\n        pet = self.create_pet()\n        pet_id = self.get_json_value(pet, \"id\")\n        data = {\n            \"id\": pet_id,\n            \"name\": self.fake.first_name(),\n            \"photoUrls\": [\"string\"],\n            \"status\": \"wrong\"\n        }\n    \n        response = MyRequests.put(\n            \"/pet/\",\n            data)\n    \n>       Assertions.assert_code_status(response, 405)\n\ntests/test_update_pet.py:148: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, expected_status_code = 405\n\n    @staticmethod\n    @allure.step(\"Check status code is {expected_status_code}\")\n    def assert_code_status(response: Response, expected_status_code):\n>       assert response.status_code == expected_status_code, \\\n            f\"Unexpected status code! Expected: {expected_status_code}. Actual: {response.status_code}\"\nE       AssertionError: Unexpected status code! Expected: 405. Actual: 200\n\nlib/assertions.py:30: AssertionError"}, "steps": [{"name": "Create pet", "status": "passed", "steps": [{"name": "Prepare pet creation data", "status": "passed", "start": 1744657923503, "stop": 1744657923503}, {"name": "POST request to URL '/pet/'", "status": "passed", "start": 1744657923503, "stop": 1744657923511}], "start": 1744657923503, "stop": 1744657923511}, {"name": "PUT request to URL '/pet/'", "status": "passed", "start": 1744657923512, "stop": 1744657923517}, {"name": "Check status code is 405", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 405. Actual: 200\n", "trace": "  File \"/Users/miowch/.pyenv/versions/n26_nastia_kuzina/lib/python3.10/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/Users/miowch/Documents/GitHub/nastia-kuzina/Task3/api-e2e-tests/lib/assertions.py\", line 30, in assert_code_status\n    assert response.status_code == expected_status_code, \\\n"}, "parameters": [{"name": "response", "value": "<Response [200]>"}, {"name": "expected_status_code", "value": "405"}], "start": 1744657923517, "stop": 1744657923517}], "start": 1744657923503, "stop": 1744657923517, "uuid": "75fa264f-f17b-478a-9f6b-b3575ddce661", "historyId": "47681c3142b156f8a36722e9b3c46245", "testCaseId": "47681c3142b156f8a36722e9b3c46245", "fullName": "tests.test_update_pet.TestUpdatePet#test_update_pet_with_invalid_status", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_pet"}, {"name": "subSuite", "value": "TestUpdatePet"}, {"name": "host", "value": "iMac.local"}, {"name": "thread", "value": "58784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_pet"}]}